### PR TITLE
Updated scripts and documentation for mirrors

### PR DESCRIPTION
**Pull request by wido**
This PR is to add documentation about the newly set up mirrors for Ceph.

This should allow users to use a local mirror instead of download.ceph.com for getting the latest and greatest Ceph.

### CODE REVIEW

File: doc/install/mirrors.rst [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+=========
+
+These mirrors are available on the following locations:
+
+- **EU: Netherlands**: http://eu.ceph.com/
+- **AU: Australia**: http://au.ceph.com/
+- **CZ: Czech Republic**: http://cz.ceph.com/
+- **SE: Sweden**: http://se.ceph.com/
+- **DE: Germany**: http://de.ceph.com/
+- **HK: Hong Kong: http://hk.ceph.com/
```
Comment (tchaikov) [inline doc/install/mirrors.rst]: missing closing `**`

**Comment by wido** [comment]
Thank you @tchaikov for the quick reply. Should be fixed.

**Comment by wido** [comment]
Can we get this one merged? I'd like to get these mirrors out there

File: doc/install/mirrors.rst [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+- Choose a mirror close to you
+- Do not sync in a interval shorter than 3 hours
+- Avoid syncing at minute 0 of the hour, use something between 0 and 59
+
+
+Becoming a mirror
+=================
+
+If you want to provide a public mirror for other users of Ceph you can opt to
+become a official mirror.
```
Comment (tchaikov) [inline doc/install/mirrors.rst]: s/a/an/

File: doc/install/mirrors.rst [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+To make sure all mirrors meet the same standards some requirements have been
+set for all mirrors. These can be found on `Github`_.
+
+If you want to apply for a official mirror, please contact the ceph-users mailinglist.
```
Comment (tchaikov) [inline doc/install/mirrors.rst]: s/a/an/

**Comment by wido** [comment]
@tchaikov Thanks! Fixed that as well.

**Pull request merged**